cli-commands/util.js

```diff
@@ -19,6 +19,7 @@ const printReportTable = (recordActions) => {
 			{ 'Status': `${getReadableStatus(action.status)}` },
 			{ 'Gas Price': `${ethersUtils.formatUnits(action.gasPrice, 'gwei')} Gwei` },
 			{ 'Gas Used': `${action.gasUsed}` },
+			{ 'NetworkID': `${action.networkID}`},
```

Comment:
What about old records that do not have networkID. It would say undefined but it is better to say something more meaningul